delta/delta_1.6.1_dss_rename_xxx_fk_overflow_characteristic_to_fk_overflow_char.sql

```diff
@@ -0,0 +1,156 @@
+-- drop connected views not needed as already listed in https://github.com/QGEP/datamodel/blob/master/view/drop_views.sql
+
+-- class overflow
+--1. add correct fk_overflow_char
+ALTER TABLE IF EXISTS qgep_od.overflow ADD COLUMN fk_overflow_char varchar (16);
```

Comment:
Why not `ALTER TABLE IF EXISTS qgep_od.overflow RENAME COLUMN fk_overflow_characteristic TO fk_overflow_char`?

Reply:
@cymed When I remember right this does not work because of the relation - you cannot rename a relation, you have to add the new one, copy the data and then delete the old one.